man/nvim.1

```diff
@@ -395,7 +395,7 @@ configuration directory.
 See
 .Ev XDG_CONFIG_HOME
 above.
-.It Pa /etc/nvim/nvimrc
+.It Pa $VIM/sysinit.vim
 The system-global
 .Nm
 configuration file.
```

Comment:
While here, perhaps we should remove the `/usr/share/nvim` reference. It's actually `/usr/local/share/nvim` on many (most?) systems and is probably not useful to most users, unlike the configuration file paths.

We could probably have CMake do some string substitution at install time so the path PREFIX is correct, but that seems excessive.